gocourse07/birdaviary/internal/storage/storage.go

```diff
@@ -0,0 +1,31 @@
+package storage
+
+import (
+	"sync"
+
+	"birdaviary/internal/logger"
+)
+
+type Storage struct {
+	data map[string][]int
+	mu   sync.Mutex
+	log  *logger.Logger
```

Comment:
Є така ідіома, як "Mutex Hat" https://dmitri.shuralyov.com/idiomatic-go#mutex-hat:
```suggestion
	mu   sync.Mutex
	data map[string][]int

	log  *logger.Logger
```